src/newViews/Tagging/Sources/BulkUpload/FilesUploaded/index.tsx

```diff
@@ -32,15 +33,22 @@ function FilesUploaded(props: Props) {
         files = [],
     } = props;
 
+    const [selectedFileId, setSelectedFileId] = useState<number | undefined>();
+    const [searchText, setSearchText] = useState<string | undefined>();
+
+    const handleSelection = useCallback((id: number) => {
+        setSelectedFileId(id);
+    }, []);
```

Comment:
Why create this separate function

Reply:
@tnagorra react warns when it is passed directly